lib/std/build.zig

```diff
@@ -1438,6 +1444,8 @@ pub const LibExeObjStep = struct {
         return run_step;
     }
 
+    /// Override the `--linker-script` argument.
```

Comment:
Rather than talking about the corresponding CLI flags, I think the documentation for the build system should explain what things do in full detail. Users of the CLI would potentially refer to the zig build docs for more information on what CLI flags do.